utilities/transactions/transaction_base.cc

```diff
@@ -451,6 +452,32 @@ Iterator* TransactionBaseImpl::GetIterator(const ReadOptions& read_options,
                                           &read_options);
 }
 
+Status TransactionBaseImpl::PutEntityImpl(ColumnFamilyHandle* column_family,
+                                          const Slice& key,
+                                          const WideColumns& columns,
+                                          bool do_validate,
+                                          bool assume_tracked) {
+  {
+    constexpr bool read_only = false;
+    constexpr bool exclusive = true;
+    const Status s = TryLock(column_family, key, read_only, exclusive,
```

Comment:
nit but I like it shorter this way :) - Just to be consistent with other places.
```
const  Status s = TryLock(column_family, key, false /* read_only */,
                     true /* exclusive */, do_validate, assume_tracked);
```